cmd/cmd.go

```diff
@@ -14,6 +14,8 @@ const (
 	FlagDatabaseDriver  = "database-driver"
 	FlagDatabaseDSN     = "database-dsn"
 	FlagGRPCAddr        = "grpc-addr"
+	FlagSSHAddr         = "ssh-addr"
```

Comment:
These should be added in alphabetical order. However, I'm going to do a major refactor of these flags, so it should be a big problem.